commands/errors.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"strings"
+)
+
+const (
+	// NoTLSWarn Warning thrown when no SSL/TLS is used
+	NoTLSWarn = "WARNING! Communication is not secure, please consider using HTTPS. Letsencrypt.org offers free SSL/TLS certificates."
+)
+
+// checkTLSInsecure checks if the given gateway has https. Use tsInsecure to skip validations
+// Note: This function does not stop the flow only prints a warning
```

Comment:
This note is slightly inaccurate, it does not actually print anything.

Reply:
I'd suggest:

// checkTLSInsecure will return a warning if the gateway URL is not using TLS and the
// tlsInsecure flag is set to false.